testing/scenario_app/ios/Scenarios/ScenariosTests/FlutterViewControllerTest.m

```diff
@@ -6,6 +6,16 @@
 #import <XCTest/XCTest.h>
 #import "AppDelegate.h"
 
+@interface FlutterViewControllerFirstFrameOverride : FlutterViewController
+@property(nonatomic) BOOL firstFrameRendered;
```

Comment:
Then this would become `@property(nonatomic, readwrite) BOOL firstFrameRendered;`